src/cpp/modules/tapenade/gmm/gmm_d.h

```diff
@@ -3,21 +3,22 @@
 #include <cstdlib>
 #include <cmath>
 
-#include "../../shared/defs.h"
+#include "../utils/adBuffer.h"
+#include "../../../shared/defs.h"
 
-// Jacobian of GMM
-void gmm_objective_d(
+// GMM function deffirintiated in reverse mode by Tapenade.
+void gmm_objective_b(
     int d,
     int k,
     int n,
     const double* const alphas,
```

Comment:
I hadn't noticed before, but these decls are a bit unwieldy.  Prefer ``double const* alphas``

Reply:
Changed in all places in tapenade submodule